simpleflow/execute.py

```diff
@@ -245,6 +248,7 @@ def execute(*args, **kwargs):
 
         # Not automatically assigned in python < 3.2.
         execute.__wrapped__ = func
+        execute.add_context_in_kwargs = True
```

Comment:
I think it breaks backwards compatibility, because it assumes that the wrapped task is ready to receive an extra kwarg. Can we set it to "False" instead and have user code actively enable it if needed? What do you think?

Reply:
It's deeply linked to https://github.com/botify-labs/simpleflow/pull/322/files#diff-5e6a5a378e912c39001ea53b61ba64edR81 I don't see how it could break anything

Reply:
OK!